preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java

```diff
@@ -22,9 +22,7 @@
  * @author Marco Sanfratello
  */
 public class Category {
-
-  private static final Logger LOGGER =
-      LogManager.getLogger(Category.class.getName());
+  private static final Logger LOGGER = LogManager.getLogger(Category.class.getName());
```

Comment:
Again, this violates the Google Java Code Style Guidelines

Reply:
how so?

Reply:
Lines may not be longer than 100 characters: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit